src/main/java/frc/robot/subsystems/armevator/Armevator.java

```diff
@@ -133,6 +135,8 @@ public Armevator() {
     elevatorMap.put(ArmevatorPose.ALGAE_L3_DROP, 20.0);
     elevatorMap.put(ArmevatorPose.ALGAE_L2_PLUCK, 3.0);
     elevatorMap.put(ArmevatorPose.ALGAE_L2_DROP, 4.5);
+    elevatorMap.put(
+        ArmevatorPose.ALGAE_PRE_PLUCK, 15.0); // TODO I just guessed for this value, check later
```

Comment:
Now that I think of it, we should have  an L2 and an L3 pre-pluck.
ALGAE_L2_PRE_PLUCK 45deg 0"
ALGAE_L3_PRE_PLUCK 15deg 10"